Recrutify/Recrutify.DataAccess/Repositories/ScheduleRepository.cs

```diff
@@ -56,6 +57,27 @@ public Task BulkUpdateScheduleSlotsAsync(Guid userId, IEnumerable<DateTime> newL
             return GetCollection().BulkWriteAsync(updateModelsWithNewScheduleSlot.Union(updateModelsWithRemovedUsers));
         }
 
+        public Task BulkСancelInterviewsAsync(IEnumerable<Interview> cancelledInterviews)
+        {
+            ScheduleCandidateInfo scheduleCandidateInfo = null;
+            var updateBuilder = Builders<Schedule>.Update;
+            var updateModels = cancelledInterviews.Select(i => new UpdateOneModel<Schedule>(
+                _filterBuilder.Eq(s => s.Id, i.UserId),
+                updateBuilder.Set("ScheduleSlots.$[scheduleSlots].ScheduleCandidateInfo", scheduleCandidateInfo))
```

Comment:
just write null, not necessary to use scheduleCandidateInfo

Reply:
+